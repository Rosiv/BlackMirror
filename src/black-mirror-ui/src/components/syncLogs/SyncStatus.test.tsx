// import * as React from 'react';
// import renderer from 'react-test-renderer';

// import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
// import getMuiTheme from 'material-ui/styles/getMuiTheme';
// import darkBaseTheme from 'material-ui/styles/baseThemes/darkBaseTheme';
// const customMuiTheme = getMuiTheme(darkBaseTheme);
// import SyncStatus from './SyncStatus';

// it('renders and is compatible with the snapshot - Scheduled status', () => {
//     const tree = renderer
//         .create(
//         <MuiThemeProvider muiTheme={customMuiTheme}>
//             <SyncStatus
//                 status={'Scheduled'}
//             />
//         </MuiThemeProvider>)
//         .toJSON();
//     expect(tree).toMatchSnapshot();
// });

// it('renders and is compatible with the snapshot - InProgress status', () => {
//     const tree = renderer
//         .create(
//         <MuiThemeProvider muiTheme={customMuiTheme}>
//             <SyncStatus
//                 status={'InProgress'}
//             />
//         </MuiThemeProvider>)
//         .toJSON();
//     expect(tree).toMatchSnapshot();
// });

// it('renders and is compatible with the snapshot - Done status', () => {
//     const tree = renderer
//         .create(
//         <MuiThemeProvider muiTheme={customMuiTheme}>
//             <SyncStatus
//                 status={'Done'}
//             />
//         </MuiThemeProvider>)
//         .toJSON();
//     expect(tree).toMatchSnapshot();
// });

// it('renders and is compatible with the snapshot - XXX status', () => {
//     const tree = renderer
//         .create(
//         <MuiThemeProvider muiTheme={customMuiTheme}>
//             <SyncStatus
//                 status={'XXX'}
//             />
//         </MuiThemeProvider>)
//         .toJSON();
//     expect(tree).toMatchSnapshot();
// });

// it('renders and is compatible with the snapshot - Retrieving status... status', () => {
//     const tree = renderer
//         .create(
//         <MuiThemeProvider muiTheme={customMuiTheme}>
//             <SyncStatus
//                 status={'Retrieving status...'}
//             />
//         </MuiThemeProvider>)
//         .toJSON();
//     expect(tree).toMatchSnapshot();
// });
