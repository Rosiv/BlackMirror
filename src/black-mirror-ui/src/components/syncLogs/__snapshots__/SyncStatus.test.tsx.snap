// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders and is compatible with the snapshot - Done status 1`] = `
<div
  className="sync-status-container"
>
  <svg
    className="sync-status-icon"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    style={
      Object {
        "color": "rgba(255, 255, 255, 1)",
        "display": "inline-block",
        "fill": "#2e7d32",
        "height": 24,
        "muiPrepared": true,
        "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
        "userSelect": "none",
        "width": 24,
      }
    }
    viewBox="0 0 24 24"
  >
    <path
      d="M19.35 10.04C18.67 6.59 15.64 4 12 4 9.11 4 6.6 5.64 5.35 8.04 2.34 8.36 0 10.91 0 14c0 3.31 2.69 6 6 6h13c2.76 0 5-2.24 5-5 0-2.64-2.05-4.78-4.65-4.96zM10 17l-3.5-3.5 1.41-1.41L10 14.17 15.18 9l1.41 1.41L10 17z"
    />
  </svg>
</div>
`;

exports[`renders and is compatible with the snapshot - Scheduled status 1`] = `
<div
  className="sync-status-container"
>
  <svg
    className="sync-status-icon"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    style={
      Object {
        "color": "rgba(255, 255, 255, 1)",
        "display": "inline-block",
        "fill": "#2196f3",
        "height": 24,
        "muiPrepared": true,
        "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
        "userSelect": "none",
        "width": 24,
      }
    }
    viewBox="0 0 24 24"
  >
    <path
      d="M22 5.72l-4.6-3.86-1.29 1.53 4.6 3.86L22 5.72zM7.88 3.39L6.6 1.86 2 5.71l1.29 1.53 4.59-3.85zM12 4c-4.97 0-9 4.03-9 9s4.02 9 9 9c4.97 0 9-4.03 9-9s-4.03-9-9-9zm0 16c-3.87 0-7-3.13-7-7s3.13-7 7-7 7 3.13 7 7-3.13 7-7 7zm-1.46-5.47L8.41 12.4l-1.06 1.06 3.18 3.18 6-6-1.06-1.06-4.93 4.95z"
    />
  </svg>
</div>
`;

exports[`renders and is compatible with the snapshot - XXX status 1`] = `
<div
  className="sync-status-container"
>
  <svg
    className="sync-status-icon"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    style={
      Object {
        "color": "rgba(255, 255, 255, 1)",
        "display": "inline-block",
        "fill": "#f44336",
        "height": 24,
        "muiPrepared": true,
        "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
        "userSelect": "none",
        "width": 24,
      }
    }
    viewBox="0 0 24 24"
  >
    <path
      d="M11 18h2v-2h-2v2zm1-16C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.41 0-8-3.59-8-8s3.59-8 8-8 8 3.59 8 8-3.59 8-8 8zm0-14c-2.21 0-4 1.79-4 4h2c0-1.1.9-2 2-2s2 .9 2 2c0 2-3 1.75-3 5h2c0-2.25 3-2.5 3-5 0-2.21-1.79-4-4-4z"
    />
  </svg>
</div>
`;
